python/hsml/util.py

```diff
@@ -189,3 +198,34 @@ def validate_metric_value(opt_val):
     raise TypeError(
         "Metric value is of type {}, expecting a number".format(type(opt_val))
     )
+
+
+def get_predictor_config_for_model(model: BaseModel):
+    if not isinstance(model, BaseModel):
+        raise ValueError(
+            "model is of type {}, but an instance of {} class is expected".format(
+                type(model), BaseModel
+            )
+        )
+
+    if model.framework is None:
+        return BasePredictorConfig(model_server=PREDICTOR.MODEL_SERVER_PYTHON)
+    if model.framework == "TENSORFLOW":
```

Comment:
Also here you can compare the class directly. `type(model) == TFModel` for example.

Reply:
Changed